src/rollover.py

```diff
@@ -95,7 +95,8 @@ def prompt_for_instances(ecs_instances, asg_contents, scale_down=False, sort_by=
         launch_time=dict(key=lambda i: i.launch_time, reverse=False),
     )
     sort_type = sorts[sort_by]
-    for x, instance in enumerate(sorted(ecs_instances, key=sort_type["key"], reverse=sort_type["reverse"])):
+    ecs_instances.sort(key=sort_type["key"], reverse=sort_type["reverse"])
```

Comment:
sorted() returns a new list, where sort() acts on the list in place. In this case sorted() returns a new list that is only used for printing the instances.